docs/managing_deis/ssl-endpoints.rst

```diff
@@ -22,9 +22,15 @@ Installing SSL on a Load Balancer
 ---------------------------------
 
 On most cloud-based load balancers, you can install a SSL certificate onto the load balancer
-itself. This is the recommended way of enabling SSL onto a cluster, as any communication inbound to
-the cluster will be encrypted while the internal components of Deis will still communicate over
-HTTP.
+itself. Any communication inbound to the cluster will be encrypted while the internal components
+of Deis will still communicate over HTTP.
+
+.. note::
+
+    On Amazon EC2, Deis enables the :ref:`PROXY protocol <proxy_protocol>` by default, requiring
+    installation of :ref:`SSL on the Deis routers <router_ssl>`, as described below.
+    Disable the PROXY protocol with ``deisctl config router rm proxyProtocol`` to terminate SSL
+    connections at an Amazon ELB instead.
```

Comment:
Since you cannot terminate SSL connections on the ELB when it's configured to listen for TCP connections, we should also note that the user has to switch the ELB's listening protocol back to HTTP.